dfget/core/downloader/p2p_downloader/p2p_downloader.go

```diff
@@ -229,6 +229,10 @@ func (p2p *P2PDownloader) pullPieceTask(item *Piece) (
 		TaskID: item.TaskID,
 	}
 
+	// dfget will sleep some time which between minTimeout and maxTimeout,
+	// the initial value is [50-100]
+	minTimeout := 50
```

Comment:
The sleepTime would be re-initialized again when get the next available pieces from supernode. It's unnecessary and increases the pressure of supernode. Actually, the initial sleepTime in the next pulling can be the value of previous sleepTime.

Reply:
Has been updated. 😄